bigtable/client/row_range.cc

```diff
@@ -0,0 +1,118 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "bigtable/client/row_range.h"
+
+#include <cstdlib>
+#include <iostream>
+#include <thread>
+
+#include <absl/memory/memory.h>
+
+namespace bigtable {
+inline namespace BIGTABLE_CLIENT_NS {
+
+const std::string RowRange::EMPTY_KEY = "\377\377\377";;
```

Comment:
Extra semicolon.

Reply:
Also, consider `"\xFF\xFF\xFF", I think those are more readable than octal in this case.